examples/text_to_image/train_text_to_image_lora.py

```diff
@@ -34,14 +34,14 @@
 from datasets import load_dataset
 from huggingface_hub import create_repo, upload_folder
 from packaging import version
+from peft import LoraConfig
+from peft.utils import get_peft_model_state_dict
```

Comment:
Users might not be aware of this dependency. Should we do `is_peft_available()` here? And add `peft` to `requirements.txt`?

Reply:
Ah makes sense, will do!